stack-definition.yaml

```diff
@@ -461,18 +461,14 @@ components:
 
 changeLog: |
 
-  - 🐛 Fixing Grafana dashboards that were broken since Gimlet Stack v0.4.0 aka the "2021.06-2" update
 
-  - 🍏 Ingress Nginx from version 0.47.0 to 0.49
-    Minor improvments in this one.
-
-    ❗ Ingress Nginx v1 is released, and in the next Gimlet Stack update we are going to upgrade to that.
+    ❗ We are investigating Ingress Nginx. There is upcoming changes with v1 (it is released), and in the next Gimlet Stack update we are going to upgrade to that.
     Please be prepared as it only supports Kubernetes versions >= v1.19
 
-  - 🍏 Cert Manager update to 1.5.3 from 1.4.0
+  - 🍏 Grafana update from version 6.13.5 to 6.16.9
```

Comment:
Could you document the Grafana version change here, not the chart version?
The chart version is a technicality that we want the use not care too much about.

This is the Grafana version: https://github.com/grafana/helm-charts/blob/main/charts/grafana/Chart.yaml#L4